src/test/java/unit/Group.java

```diff
@@ -315,6 +294,31 @@ private static void expectListedMemberWithPolicies(
         "test user has the right policies");
   }
 
+  private void expectListedMemberWithPoliciesTableFormat(
+      String name, String email, GroupPolicy... policies) throws JsonProcessingException {
+    // test the group list-user as a table format
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list-users", "--name=" + name);
+
+    // use regular expression testing the table format and content inside
+    assertTrue(cmd.stdErr == null || cmd.stdErr.isEmpty());
+    assertEquals(0, cmd.exitCode, "group list-user returned successfully");
+    String[] rows = cmd.stdOut.split("\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("EMAIL", rowHead[0].trim().replace("\r", ""));
+    assertEquals("POLICIES", rowHead[1].trim().replace("\r", ""));
+
+    // test the email exsit in the table and the policy correct
+    Boolean emailExist = false;
+    for (int i = 1; i < rows.length; i = i + 1) {
+      String[] rowi = rows[i].split("\\s+", 2);
+      if (email.equalsIgnoreCase(rowi[0])) {
+        assertEquals("[MEMBER, ADMIN]", rowi[1].trim().replace("\r", ""));
```

Comment:
Instead of hard-coding `"[MEMBER, ADMIN]"`, you should use the `policies` that's passed into this method.

Instead of split, I recommend just doing this, to make test easier to read.

```
    for (int i = 1; i < rows.length; i = i + 1) {
      if (row[i].contains(email)) {
        emailExists = true;
        for (var policy : policies) {
          assert row[i] contains policy
        } 
    }
```